stats/config/charts.json

```diff
@@ -65,96 +65,96 @@
             "title": "Transactions fees",
-            "description": "Amount of tokens paid as fees",
+            "description": "Amount of tokens paid as fees per period",
             "units": "{{native_coin_symbol}}"
         },
         "new_txns": {
             "title": "New transactions",
-            "description": "New transactions number"
+            "description": "New transactions number per period"
         },
         "txns_growth": {
             "title": "Transactions growth",
             "description": "Cumulative transactions number"
         },
         "txns_success_rate": {
             "title": "Transactions success rate",
-            "description": "Successful transactions rate per day"
+            "description": "Successful transactions rate per period"
         },
         "new_blocks": {
             "title": "New blocks",
-            "description": "New blocks number"
+            "description": "New blocks number per period"
         },
         "average_block_size": {
             "title": "Average block size",
-            "description": "Average size of blocks in bytes",
+            "description": "Average size of blocks per period",
             "units": "Bytes"
         },
         "average_block_rewards": {
             "title": "Average block rewards",
-            "description": "Average amount of distributed reward in tokens per day",
+            "description": "Average amount of distributed rewards in tokens per period",
             "units": "{{native_coin_symbol}}"
         },
         "new_native_coin_transfers": {
-            "title": "New {{native_coin_symbol}} transfers",
-            "description": "New token transfers number for the period"
+            "title": "Number of {{native_coin_symbol}} transfers",
+            "description": "Token transfers performed during the period"
         },
         "native_coin_holders_growth": {
             "enabled": false,
             "title": "{{native_coin_symbol}} holders growth",
-            "description": "Cumulative token holders number for the period"
+            "description": "Cumulative number of token holders"
         },
         "new_native_coin_holders": {
             "enabled": false,
             "title": "New {{native_coin_symbol}} holders",
-            "description": "New token holders number per day"
+            "description": "New token holders number per period"
         },
         "native_coin_supply": {
             "enabled": false,
             "title": "{{native_coin_symbol}} circulating supply",
-            "description": "Amount of token circulating supply for the period",
+            "description": "Amount of token circulating supply per period",
             "units": "{{native_coin_symbol}}"
         },
         "average_gas_limit": {
             "title": "Average gas limit",
-            "description": "Average gas limit per block for the period"
+            "description": "Average block gas limit per period"
```

Comment:
Average gas limit per block

Reply:
Maybe "Average block gas limit" to avoid repeating the title?